tests/containers/docker_image.pm

```diff
@@ -45,6 +45,7 @@ sub run {
     for my $i (0 .. $#$image_names) {
         test_container_image(image => $image_names->[$i], runtime => 'docker');
         build_container_image(image => $image_names->[$i], runtime => 'docker');
+        build_with_zypper_docker(image => $image_names->[$i], runtime => 'docker');
         test_opensuse_based_image(image => $image_names->[$i], runtime => 'docker');
```

Comment:
```suggestion
        test_opensuse_based_image(image => $image_names->[$i], runtime => 'docker');
        build_with_zypper_docker(image => $image_names->[$i], runtime => 'docker');
```
As you run the `test_opensuse_based_image()` also for the `zypper-docker` derived image it imo makes sense to run the `test_opensuse_based_image()` for the original image before.

Reply:
That is a good point as well :D  Thanks!